serialization-jackson/src/main/scala/org/apache/pekko/serialization/jackson/JacksonObjectMapperProvider.scala

```diff
@@ -512,6 +521,24 @@ class JacksonObjectMapperFactory {
       : immutable.Seq[(DeserializationFeature, Boolean)] =
     configuredFeatures
 
+  /**
+   * After construction of the `ObjectMapper` the configured enum features are applied to
+   * the mapper. These features can be amended programatically by overriding this method and
+   * return the features that are to be applied to the `ObjectMapper`.
+   *
+   * When implementing a `JacksonObjectMapperFactory` with Java the `immutable.Seq` can be
+   * created with [[pekko.japi.Util.immutableSeq]].
+   *
+   * @param bindingName bindingName name of this `ObjectMapper`
+   * @param configuredFeatures the list of `DateTimeFeature` that were configured in
+   *                           `pekko.serialization.jackson3.enum-features`
+   */
+  def overrideConfiguredEnumFeatures(
+      @unused bindingName: String,
+      configuredFeatures: immutable.Seq[(EnumFeature, Boolean)])
+      : immutable.Seq[(EnumFeature, Boolean)] =
+    configuredFeatures
```

Comment:
then the java api should be a Pair

Reply:
Java doesn't have a pair, right? Then why not Map?

Reply:
This code is consistent with the existing code in the class - ie there are similar methods already and this is basically a copy/paste of one of them.